### PR TITLE
RATEST-96: Fixing End Visit Test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket id: https://issues.openmrs.org/browse/RATEST-96
Fixed failing test of End Visit test in the legacy ui

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java [hunk 1]
```diff
@@ -33,7 +33,6 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
```
Comment (HerbertYiga) [inline ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java:36]: you can also remove the package for the @ignore from this file

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java:36]: Thanks @HerbertYiga

**Comment by kdaud** [comment]
Refer to this [commit](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/399)

**Comment by kdaud** [comment]
Refer to this [commit](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/399)

**Pull request merged**